src/runtime/Icon.vue

```diff
@@ -86,14 +88,14 @@ async function loadIconComponent () {
   if (component.value) {
     return
   }
-  if (!state.value?.[iconName.value]) {
+  if (!state.value?.[iconName.value.name]) {
     isFetching.value = true
-    state.value[iconName.value] = await loadIcon(iconName.value as string).catch(() => undefined)
+    state.value[iconName.value.name] = await loadIcon({ provider: '', ...iconName.value }).catch(() => undefined)
```

Comment:
This is breaking my application.

I used to be able to do this in app.config.ts:

```
export default defineAppConfig({
  nuxtIcon: {
    iconifyApiOptions: {
      url: 'https://icon.dust.rs',
      publicApiFallback: true // default: false
    }
  },
})
```
and then specify the icon as "@custom:fa6-pro-duotone:block-question"

With this change in place, I can no longer do that.

Reply:
Sorry about this @Neo-Zhixing 

I just published `v0.6.4` that fix this issue, I tested with your example and it works: 

```vue
<Icon name="@custom:fa6-pro-duotone:block-question" />
```

Reply:
Thanks! That was a quick fix.